pkg/reftracker/ref_tracker_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package reftracker
+
+import "testing"
+
+func Test_AddAppForRef_AddsApp_WhenRefNotInMap(t *testing.T) {
+	appRefTracker := NewAppRefTracker()
+	appRefTracker.AddAppForRef("secret", "secretName", "default", "app")
+
+	if _, ok := appRefTracker.refsToApps["secret:secretName:default"]["app"]; !ok {
```

Comment:
we shouldnt access private fields (refsToApps) in tests since that's couples tests with impl (also doesnt prove that class external behaviour is correct).

Reply:
Changed to accessing through appRefTracker funcs.